src/vmm/src/version_map.rs

```diff
@@ -63,6 +63,15 @@ lazy_static! {
 
     /// Static instance used for creating a 1:1 mapping between Firecracker release version
     /// and snapshot data format version.
+    /// !CAVEAT!
+    /// This map is supposed to be strictly one-to-one (i.e. bijective) because
+    /// describe-snapshot inverts it to look up the release that matches the
+    /// snapshot version. If two version map to the snap_version, the results
```

Comment:
nit:     /// snapshot version. If two versions map to the given snap_version, the results

Reply:
Nice. Will add it to my next PR.